src/entrypoint/docker-entrypoint.sh

```
@@ -380,19 +387,49 @@ populatePhpEnv() {
+        if [[ ! -r $aws_ak_file ]]; then 
+            aws_ak_file=/run/secrets/elab_aws_access_key
+            printf "%s" "$aws_ak" > "$aws_ak_file"
+        fi
+        sed -i -e "s|%ELAB_AWS_ACCESS_KEY%|${aws_ak_file}|" /etc/php84/php-fpm.d/elabpool.conf
     else
         sed -i -e "/%ELAB_AWS_ACCESS_KEY%/d" /etc/php84/php-fpm.d/elabpool.conf
+    fi
+    
+    if [[ -n $aws_sk || -n $aws_sk_file ]]; then
+        if [[ ! -r $aws_sk_file ]]; then 
+            $aws_sk_file=/run/secrets/elab_aws_secret_key
```

Shellcheck (0.11.0)

[error] Don't use $ on the left side of assignments.

(SC2281)